meinberlin/assets/scss/components/_status_bar.scss

```diff
@@ -33,8 +33,8 @@
 
 .status-bar__past {
     padding: $padding;
-    background-color: $brand-secondary;
-    color: contrast-color($brand-secondary);
+    background-color: $blue-light;
+    color: $print-black;
```

Comment:
This is a colour, we use for the printing media-query. Isn't there another black we use everywhere else? Like $text-colour? Alright, I checked that. So, we are sure, that this should use another bg and another text-colour than everywhere else on the site?

Reply:
Just say yes, we are very sure and I will merge! :)

Reply:
Cool! Which header? The berlin-one? It should be in variables already then! Maybe bg-secondary or bg-tertiary?